includes/Gateway/Gift_Card.php

```diff
@@ -319,7 +319,7 @@ public function add_gift_card_fragments( $fragments ) {
 					<td><?php echo wc_price( $cart_total - $response['difference'], array( 'currency' => get_woocommerce_currency() ) ); ?></td>
 				</tr>
 				<tr>
-					<td><?php esc_html_e( 'Credit card', 'woocommerce-square' ); ?></td>
+					<td><?php esc_html_e( 'Credit card/Cash App Pay', 'woocommerce-square' ); ?></td>
```

Comment:
Do we want to hardcode Cash App Pay here and in the line below? I guess wondering what happens if someone doesn't have Cash App Pay enabled? I'm assuming this message will still show which may be confusing.

Reply:
Nice catch! addressed in https://github.com/woocommerce/woocommerce-square/pull/70/commits/38adc7d744a5c3fa01ccec4bfc453440b5aa17dd